components/molecules/List.js

```diff
@@ -0,0 +1,43 @@
+import PropTypes from "prop-types";
+
+/**
+ *  List component
+ */
+export function List(props) {
+  let opacity = 20;
+  return (
+    <ol className={props.className}>
+      {props.items.map((item, key) => {
+        opacity += 20;
```

Comment:
Opacity is gonna start at 40 this way. Assign a variable to store the JSX and then add 20 to the opacity after. Note if the opacity === 100 .... 20 should not be added

Reply:
The opacity in Figma design starts at 40 that's why I left it that way. Is it still better to put the JSX in a variable?